### ISSUE TITLE
float type

### ISSUE THREAD

**Issue by redrobotdev** [JackAdams/meteor-editable-text#81]
with current implementation you cannot save the value as float

**Comment by JackAdams** [comment]
Yes, that's right.  It's not something I've ever needed, so I haven't implemented it.

The code base is a bit messy but if you want to do a PR I'll happily accept it.

**Comment by redrobotdev** [comment]
I can take a look at it at spare time. Though a quick solution would be to use beforeInsert callback and convert from string to float correct? I haven't been able to get the callback to trigger. 

in the blaze template:
```
{{> editableText context=.. collection="AudioFiles field=(freq @index) acceptEmpty=true unsetEmpty=true **beforeInsert="convertToFloat"** substitute='<i class="fa fa-pencil"></i>'}}
```

In the template .js controller
```
EditableText.registerCallbacks({
    convertToFloat : function (documentToBeInserted, Collectiondoc) {
        console.log("convert to float");
        return true;
    },
});

```

**Comment by JackAdams** [comment]
Yes, that would work, but use `beforeUpdate` rather than `beforeInsert`. (The widget only inserts a whole document if it has `autoInsert=true` set and a context that doesn't contain an `_id` value. Otherwise it just updates the particular field value.)
